flake.nix

```diff
@@ -27,8 +27,8 @@
     advisory-db.flake = false;
     risc0pkgs.url = "github:cspr-rad/risc0pkgs";
     risc0pkgs.inputs.nixpkgs.follows = "nixpkgs";
-    csprpkgs.url = "github:cspr-rad/csprpkgs/add-cctl";
-    csprpkgs.inputs.nixpkgs.follows = "nixpkgs";
+    cctl.url = "github:casper-network/cctl/fix-macos";
```

Comment:
Why `fix-macos` branch? I see that there is still [concern about it failing on macOS](https://github.com/casper-network/cctl/pull/12#pullrequestreview-2034158483).

Reply:
The pre-installed `cp` command on MacOS does not support `--no-preserve=mode` in the Nix work I packaged it with GNU's `coreutils` which contains the proper `cp` command. Unfortunately there is no alternative for the MacOS binary. This is subject to discussions. If we don't use `--no-preserve=mode` we run into permission problems, since the chainspec files come from the Nix store, and those files are owned by root. It would be good to find a way to support both Nix and otherwise vendored tooling.

Reply:
They are not only owned by root but also read-only. but we need write permissions too.

Reply:
With recent `chmod` fix in https://github.com/casper-network/cctl/pull/12/commits/947c34b991e37476db82ccfa2bd7c0312c1a91d7 I am happy with changes.